pkgs/servers/dns/knot-resolver/default.nix

```diff
@@ -114,7 +114,8 @@ wrapped-full = runCommand unwrapped.name
       # For http module, prefill module, trust anchor bootstrap.
       # It brings lots of deps; some are useful elsewhere (e.g. cqueues).
       http
-      # psl isn't in nixpkgs yet, but policy.slice_randomize_psl() seems not important.
+      # used by policy.slice_randomize_psl()
```

Comment:
is slice_randomize_psl usage  tested ?I've broken knot-resolver a few times recently so the more we test it, the less worried I am about breaking @vcunat 's favortie software :)

Reply:
I think if http is working, this is also working.

Reply:
Sounds safe enough to me, but longterm future/maintenance of PSL stuff seems unclear.